### DISCUSSION TITLE
Outlet not function :/

### DISCUSSION THREAD

**Discussion by efgomes** [TanStack/router#2145] [General]
![image](https://github.com/user-attachments/assets/c2ec0704-803b-4a18-b23d-738c0405320a)

"I need my route $pipelineId to be nested within virtual-reception, I can access the routes, but they are not nested."

**Comment by efgomes** [comment]
![image](https://github.com/user-attachments/assets/e8e728c7-1dcf-455a-a916-dcf48b031d57)
The only way I've been able to make it work so far is by duplicating the files. I keep one named 'index' and another named '_header'. This way I can access /crm, and when there is an action, I will navigate to crm/
, but in a nested manner. My question is whether this is being done correctly, and I would like to improve the code and do it in the way that the documentation recommends.

Thank you."

**Reply by schiller-manuel** [reply]
As it stands, your questions is unclear to me. Can you please provide a complete minimal example by forking one of the existing router example (see e.g. https://tanstack.com/router/v1/docs/framework/react/examples/quickstart-file-based)

Aside from that, I **guess** you need to create a `/routes/virtual-reception/route.tsx` OR `/routes/virtual-reception.tsx` which then renders the outlet. This will then contain either the index route or the `$pipelineId` route.

**Reply by efgomes** [reply]
![image](https://github.com/user-attachments/assets/a30777a3-999b-4dd6-b0ab-3c5959269c99)
"I'll try to be clearer, I removed the previous routes and now it's set up this way: I created the 'index', and inside it there is an 'Outlet', so we expect everything that is rendered at /crm/4 to be rendered along with the content of the index. However, this does not happen, it only renders the content of $pipelineId.tsx."

**Reply by schiller-manuel** [reply]
as pointed out above, please provide a minimal complete example instead of screenshots
